R/predict.R

```diff
@@ -72,6 +72,26 @@ predict.WrappedModel = function(object, task, newdata, subset = NULL, ...) {
     newdata = newdata[subset, , drop = FALSE]
   }
 
+
+  # set factor levels, present in test but missing in train, to NA
+  if (model$learner$fix.factors.prediction == TRUE &&
+      any(vcapply(newdata, function(x) class(x)) == "factor") &&
+      any(class(model$learner.model) == "lm" | class(model$learner.model) == "glmmPQL")) {
```

Comment:
Other models than those inheriting `lm` seem to work without any problems in cases addressed by this PR. I would prefer not to change anything in the data if everything is working smoothly as this may cause unwanted side-effects. 

Doing it globally here for any models that inherit `lm` is way cheaper than including/duplicating this in all `lm`* learners? I will remove `glmmPQL` again as it is not a supported learner atm